ouroboros-consensus/src/Ouroboros/Consensus/NodeKernel.hs

```diff
@@ -565,13 +565,18 @@ forkBlockForging IS{..} blockForging =
         trace $ TraceNodeIsLeader currentSlot
 
         -- Tick the ledger state for the 'SlotNo' we're producing a block for
-        let tickedLedgerState :: Ticked (LedgerState blk)
-            tickedLedgerState =
+        tickedLedgerState :: Ticked (LedgerState blk) <-
+            lift . evaluate $ -- TODO is evaluate enough here?
               applyChainTick
                 (configLedger cfg)
                 currentSlot
                 (ledgerState unticked)
 
+        let newBlockSlot = case getTipSlot tickedLedgerState of
```

Comment:
This should be the tip of `unticked`, not `tickedLedgerState`.

Reply:
Actually, just logging `bcPrevPoint` here would be fine.

Reply:
We chose to log `bcPrevPoint` instead of extracting the tip slot from `unticked`.